docs/Configuration.md

```diff
@@ -253,13 +278,15 @@ The ZONEMASTER section has several keys :
 
 ### max_zonemaster_execution_time
 
-An integer.
+A positive integer. Max length 9 digits.
+
 Time in seconds before reporting an unfinished test as failed.
 Default value: `600`.
 
 ### maximal_number_of_retries
 
-An integer.
+A non-negative integer. Max length 9 digits.
```

Comment:
Until we get a request for larger number, I think 1 digit is enough.

Reply:
I'll take that :)